distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Match_2.enso

```diff
@@ -53,15 +62,176 @@ type Match_2
 
        In this case, the group id for "(d)", which is 3, is a valid group id and
        (Pattern_2.lookup_group 3) will return 3. If the caller tries to get group 3,
-       Match_2.group will return Nothing.
-    group : Integer | Text -> Span
-    group self id =
+       Match_2.group will return the default value.
+    span : Integer | Text -> Any | Nothing -> Utf_16_Span ! No_Such_Group
+    span self id=0 default=Nothing =
```

Comment:
Let`s rename to `utf_16_span` and make it explicit.

We should also rename `id` to `group`.